site/_posts/2022-07-15-remix-demo.md

```diff
@@ -85,13 +86,13 @@ Here's an example of the `.env` file, with values filled out. Some of your value
 ```
 CLIENT_ID="85a03867-dccf-4882-adde-1a79aeec50df"
 CLIENT_SECRET="b4xOdsBUWHQkkU3BOqAxSilfttI4TJv9eI_LOj8zVgE"
-AUTH_URL="http://localhost:9000"
+AUTH_URL="http://localhost:9011/oauth2"
```

Comment:
please also pull in any code file using `remote_include` from the remix example app.

Reply:
I used `remote_include` for all other code. The example `.env` like this is not in the remix repository.

Reply:
We can pull in https://github.com/FusionAuth/fusionauth-example-remix/blob/main/env.example.

Reply:
nm